app/src/main/java/com/artemzin/qualitymatters/ApplicationModule.java

```diff
@@ -33,6 +34,11 @@ public QualityMattersApp provideQualityMattersApp() {
     }
 
     @Provides @NonNull @Singleton
+    public Context provideContext() {
```

Comment:
It's because ```Paperwork``` constructor accepts a ```Context``` object, and Dagger will not compile if there's no  ```@Provides``` for that. I cannot use the application object.